js/controller.js

```diff
@@ -411,6 +408,9 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
     },
 
     onVcPlayed: function(event, source) {
+      var language = "";
```

Comment:
As part of our intention here to wipe out the data, we set the language to a blank entry so it doesn't display on the endscreen.